docs/docs/user/publishing.md

```diff
@@ -0,0 +1,155 @@
+# Publishing
+
+Simplest way to start publishing to Hermes is sending POST request on topic endpoint::
+
+```
+curl -X POST -H "Content-Type: application/json" http://hermes-frontend/topics/group.topic -d '{"message": "Hello world!"}'
+```
+
+## Creating group
+
+As [data model](/overview/data-model) describes, topics are gathered into topic groups. If you don't have a group yet,
+you need to create one by sending POST with `application/json` content type at
+
+```
+/groups
+```
+
+Body of request must contain:
+
+* name: group name
+* supportTeam: team that should have administrative access to this group
+
+Minimal request:
```

Comment:
Minimal request is represented in quickstart section, maybe it's good place to describe all Group fields?